frontend/src/lib/components/modals/FormModal.svelte

```diff
@@ -77,8 +77,12 @@
     <slot name="extraActions" />
   </svelte:fragment>
   <svelte:fragment slot="actions" let:closing>
-    <button type="submit" form="modalForm" class="btn btn-primary" class:loading={closing}>
-      <slot name="submitText" />
+    <button type="submit" form="modalForm" class="btn btn-primary">
+      {#if !closing}
+        <slot name="submitText" />
+      {:else}
+        <span class:loading={closing}></span>
+      {/if}
```

Comment:
the way this is setup the submit text will not display when we show loading, which isn't how it used to work. Previously it would show the spinner before the text.